src/server/jenkinsApi.js

```diff
@@ -2,130 +2,166 @@ const express = require('express');
 const router = express.Router();
 const axios = require('axios');
 
-const QA_BASE_URL = 'http://ci.qfun.com:8080/job/pureconnect/job/interaction_connect/job/connect_cic_regression';
-const MAIN_BASE_URL = 'http://ci.qfun.com:8080/job/pureconnect/job/interaction_connect/job/connect_main_regression';
-const JSON_API = '/api/json';
-const LAST_SUCCESSFUL_ROUTE = '/lastSuccessfulBuild/testReport';
-
-const QA_JOB_URL_API = QA_BASE_URL + JSON_API;
-const QA_LAST_SUCCESSFUL_TEST_REPORT_URL = QA_BASE_URL + LAST_SUCCESSFUL_ROUTE;
-const QA_LAST_SUCCESSFUL_TEST_REPORT_URL_API = QA_BASE_URL + LAST_SUCCESSFUL_ROUTE + JSON_API;
+const WEBIC_ICAT_URL = 'http://ci.qfun.com:8080/job/pureconnect/job/interaction_connect/job/webic-icat/api/json'
+const MAIN_BASE_URL = 'http://ci.qfun.com:8080/job/pureconnect/job/interaction_connect/job/webic-icat/job/client.test.latest_systest/';
```

Comment:
Its probably worth while to break these variables up more.  There are a number of things that will be potentially changing.  So things I would have consts:

1. http://ci.qfun.com:8080
2. job/pureconnect/job/interaction_connect
3. webic-icat (I am not sure we will be keeping this for QA streams)
4. client.test.latest_systest